packages/dashboard/src/components/run/details.tsx

```diff
@@ -27,25 +36,120 @@ export function RunDetails({ run }: RunDetailsProps) {
+              render: spec => {
+                if (spec?.results?.stats?.wallClockDuration) {
+                  return (
+                    <Tooltip text={`Started at ${spec.results.stats.wallClockStartedAt}`}>
+                      <Text>{shortEnglishHumanizerWithMsIfNeeded(spec.results.stats.wallClockDuration, miniutesAndSecondsOptions)}</Text>
+                    </Tooltip>
+                  )
+                } else if (spec.claimedAt) {
+                  return (
+                    <Tooltip text={`Started at ${spec.claimedAt}`}>
+                      <Text>
+                        <RenderOnInterval live refreshIntervalInSeconds={1} renderChild={()=>{
+                          return `${shortEnglishHumanizerWithMsIfNeeded(new Date() - new Date(spec.claimedAt), miniutesAndSecondsOptions)}`
+                        }} />
+                      </Text>
+                    </Tooltip>
+                  )
+                } else {
+                  return '';
                 }
-              `}
-            >
-              <SpecSummary spec={spec} />
-            </li>
-          ))}
-      </ul>
+              },
+            },
+            {
+              name: 'failures',
+              header: '',
+              sortable: false,
+              render: spec => (
+                spec?.results?.stats?.failures ?
+                <Text color="danger">
+                  <Tooltip text='failed'>
+                    <span className={centeredIconClassName}>
+                      <Icon
+                        icon="exclamationTriangleOutline"
+                        size={1}
+                      />
+                      {spec.results.stats.failures}
+                    </span>
+                  </Tooltip>
+                </Text> : ''
+              )
+            },
+            {
+              name: 'passes',
+              header: '',
+              sortable: false,
+              render: spec => (
+                spec?.results?.stats?.passes ?
+                <Text color="success">
+                  <Tooltip text='passed'>
+                    <span className={centeredIconClassName}>
+                      <Icon
+                        icon="checkCircleOutline"
+                        size={1}
+                      />
+                      {spec.results.stats.passes}
+                    </span>
+                  </Tooltip>
+                </Text> : ''
+              )
+            },
+            {
+              name: 'skipped',
+              header: '',
+              sortable: false,
+              render: spec => (
+                spec?.results?.stats?.pending ?
+                <Text>
+                  <Tooltip text='skipped'>
+                    <span className={centeredIconClassName}>
+                      <Icon
+                        icon="timesOutline"
+                        size={1}
+                      />
+                      {spec.results.stats.pending}
```

Comment:
I didnt look into this but its super odd that the pending stat is actually used for skipped.